README.md

```diff
@@ -57,3 +57,16 @@ If you have any questions or ideas feel free to join our <a href="https://discor
 # The license
 
 Bopis app is completely free and released under the Apache v2.0 License. Check <a href="https://github.com/hotwax/bopis/blob/main/LICENSE" target="_blank">LICENSE</a> for more details.
+
+<h2>Project Contributors⭐</h2> 
+
+  <a name = "contributors"></a>
```

Comment:
We might not need it if we used markdown
```suggestion
```

Reply:
@adityasharma7 I've Made the changes